allspice/utils/bom_generation.py

```diff
@@ -34,6 +36,87 @@ class VariationKind(Enum):
 Bom = list[BomEntry]
 
 
+def generate_bom(
+    allspice_client: AllSpice,
+    repository: Repository,
+    project_tool: Literal["altium", "orcad"],
```

Comment:
There is a possibility that we can automatically infer this from the repo using https://hub.allspice.io/api/swagger#/repository/repoGetLanguages